src/components/Calendar/Calendar.tsx

```diff
@@ -30,22 +32,40 @@ function Calendar({
   variant = 'default',
   subtitle = '',
 }: Props) {
-  const current = date || dayjs();
+  const [current, setCurrent] = useState(dates.length > 0 ? dates[0] : dayjs());
   const weeks = getWeeks(current.year(), current.month());
 
+  const select = (date: dayjs.Dayjs) => {
+    if (type === 'single') {
```

Comment:
what happens if the user passes a type that is neither range nor single. it could be that the selection stops working then.

I would suggest that default (no matter what the user passes for type) should always be single.